app/main/posts/views/post-view-data.controller.js

```diff
@@ -92,7 +92,7 @@ function PostViewDataController(
     // filters on the frontend. This makes it MUCH faster than using the API
     $rootScope.$on('event:edit:post:status:data:mode:saveSuccess', function (event, args) {
```

Comment:
Forgot to note... I know this was here already, but its a little worrying listening on $rootScope because it won't get unbound when navigating away.
Does this still work if we use `$scope.$on`? Thats much safer, but I can't remember if it'll see the event on a directive scope.